patterns/home.php

```diff
@@ -6,7 +6,8 @@
  * Inserter: no
  */
 ?>
-<!-- wp:group {"style":{"spacing":{"padding":{"top":"6vh","bottom":"6vh"}}},"layout":{"type":"constrained"}} -->
+<!-- wp:group {"tagName":"main","layout":{"type":"default"}} -->
+<main class="wp-block-group"><!-- wp:group {"style":{"spacing":{"padding":{"top":"6vh","bottom":"6vh"}}},"layout":{"type":"constrained"}} -->
```

Comment:
Can we edit the group instead that is existing instead of adding another one?

Reply:
That existing group is only for the first row of content. The new group puts all the non-header and non-footer content into the "main" landmark.

Reply:
Just noting that landmarks aren’t necessary in patterns, but they are in templates. Patterns are inserted many times over; we can assume users won’t have multiple patterns (i.e. multiple main tags).